tests/integration/security/sds_ingress/ingress_test.go

```diff
@@ -460,6 +460,7 @@ func TestMultiTlsGateway_InvalidSecret(t *testing.T) {
 func TestMultiMtlsGateway_InvalidSecret(t *testing.T) {
 	framework.
 		NewTest(t).
+		Features("security.ingress.mtls").
```

Comment:
same here, you have `security.ingress.mtls.gateway` for TestMtlsGateways, but here it's just `security.ingress.mtls` for TestMultiMtlsGateway_InvalidSecret, this looks very inconsistent.

Reply:
The feature label framework won't allow non-leafs from the yaml tree. This has to have something after mtls.

Reply:
change it